### PR TITLE
Optimize `ALTER INSTANCE  SET XA_RECOVERY_ID`

### PR DESCRIPTION
**Pull request by lanchengx**
For https://github.com/apache/shardingsphere/issues/15405

Changes proposed in this pull request:
- change XA_RECOVERY_ID to `XA_RECOVERY_NODES`
- Update parse test

### CODE REVIEW

File: shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RALStatement.g4 [hunk 1]
```diff
@@ -171,6 +171,10 @@ variableName
     : IDENTIFIER
     ;
 
+variableValues
+    : variableValue (COMMA variableValue)*?
+    ;
+
```
Comment (jingshanglu) [inline shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RALStatement.g4:177]: Maybe It is ok like this
```
variableValues
    : variableValue (COMMA variableValue)*
    ;
```

File: shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml [hunk 2]
```diff
@@ -52,6 +52,6 @@
     <distsql-case id="prepare-distsql" value="PREPARE DISTSQL" />
     <distsql-case id="apply-distsql" value="APPLY DISTSQL" />
     <distsql-case id="discard-distsql" value="DISCARD DISTSQL" />
-    <distsql-case id="alter-instance-xa-recovery-id" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_ID=ID_TEST" />
+    <distsql-case id="alter-instance-xa-recovery-nodes" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308" />
```
Comment (jingshanglu) [inline shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml:55]: It maybe better to add a case like
```
ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308,1.2.3.4@3307;
```

File: shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RALStatement.g4 (hunk 1 above)
Comment (lanchengx) [reply shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RALStatement.g4:177]: Right!

File: shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml (hunk 2 above)
Comment (lanchengx) [reply shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml:55]: it was provided to openForce https://github.com/apache/shardingsphere/issues/16357

**Review by jingshanglu — APPROVED** [review]

**Pull request merged**